docs/resources/Channel.md

```diff
@@ -244,6 +245,12 @@ Represents a message sent in a channel within Discord.
 | LISTEN | 3 |
 | JOIN_REQUEST | 5 |
 
+###### Message flags
+
+| Bit (0-based) | Bit value | Description |
```

Comment:
I think it would be better if these would be shown like here: https://discordapp.com/developers/docs/resources/user#user-object-user-flags

Reply:
Good idea. I wasn't sure how to best do the table anyway